src/datepicker/DatePickerState.ts

```diff
@@ -55,15 +56,10 @@ export const useDatePickerState = (props: DatePickerInitialState = {}) => {
     shouldUpdate: (prev, next) => prev !== next,
   });
 
-  const dateValue = value && isValid(value) ? new Date(value) : undefined;
-  const minValue =
-    minValueProp && isValid(minValueProp) ? new Date(minValueProp) : undefined;
-  const maxValue =
-    maxValueProp && isValid(maxValueProp) ? new Date(maxValueProp) : undefined;
-  const placeholderDate =
-    placeholderDateProp && isValid(placeholderDateProp)
-      ? new Date(placeholderDateProp)
-      : undefined;
+  const dateValue = convertValue(value);
+  const minValue = convertValue(minValueProp);
+  const maxValue = convertValue(maxValueProp);
+  const placeholderDate = convertValue(placeholderDateProp);
```

Comment:
These are changed aswell in #88

Reply:
We will fix these issues later. after merging. no need to worry now. just wanna get the core logic done